### PR TITLE
Bug when opening User Class

### PR DESCRIPTION
**Pull request by fn-faisal**
Fix the bug related to array of pointers.

### CODE REVIEW

File: src/components/BrowserCell/BrowserCell.react.js [hunk 1]
```diff
@@ -250,11 +251,19 @@ class BrowserCell extends Component {
       this.copyableValue = value.id;
     }
     else if (type === 'Array') {
-      if ( typeof value[0] === 'object' && value[0].__type === 'Pointer' ) {
+      if ( value[0] && (typeof value[0] === 'object' && value[0].__type === 'Pointer') ) {
         const array = [];
+        let mapError = false;
         value.map( (v, i) => {
+          if ( !v || v === null || v === undefined ) {
```
Comment (davimacedo) [inline src/components/BrowserCell/BrowserCell.react.js:258]: We don't need the second and third verification

File: src/components/BrowserCell/BrowserCell.react.js [hunk 2]
```diff
@@ -250,11 +251,19 @@ class BrowserCell extends Component {
       this.copyableValue = value.id;
     }
     else if (type === 'Array') {
-      if ( typeof value[0] === 'object' && value[0].__type === 'Pointer' ) {
+      if ( value[0] && (typeof value[0] === 'object' && value[0].__type === 'Pointer') ) {
```
Comment (davimacedo) [inline src/components/BrowserCell/BrowserCell.react.js:254]: The second () marks are not necessary

File: src/components/BrowserCell/BrowserCell.react.js [hunk 3]
```diff
@@ -250,11 +251,19 @@ class BrowserCell extends Component {
...
         const array = [];
+        let mapError = false;
         value.map( (v, i) => {
+          if ( !v || v === null || v === undefined ) {
+            console.error('Invalid type found in pointer array');
```
Comment (davimacedo) [inline src/components/BrowserCell/BrowserCell.react.js:259]: Could you please remove the two console.error() calls?

**Review by davimacedo — CHANGES_REQUESTED** [review]

File: src/components/BrowserCell/BrowserCell.react.js (hunk 1 above)
Comment (fn-faisal) [reply src/components/BrowserCell/BrowserCell.react.js]: I've refactored it to one line.

File: src/components/BrowserCell/BrowserCell.react.js (hunk 2 above)
Comment (fn-faisal) [reply src/components/BrowserCell/BrowserCell.react.js]: The () have been removed

File: src/components/BrowserCell/BrowserCell.react.js (hunk 3 above)
Comment (fn-faisal) [reply src/components/BrowserCell/BrowserCell.react.js]: I've removed the console.error

**Review by davimacedo — APPROVED** [review]

**Pull request merged**